### PR TITLE
Add notes about log4shell

### PR DESCRIPTION
**Pull request by renner**
This patch is to update the index and news pages with information about the log4shell vulnerability. The mailing list archive is currently outdated, so we cannot link to the announcement email (reported [here](https://progress.opensuse.org/issues/103911)).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -233,20 +233,20 @@ <h3>Latest news</h3>
           <!-- News box Start -->
           <div class="news-box">
             <div class="news-box-title">
-              <span>09/12/2021</span>
+              <span>13/12/2021</span>
             </div>
             <div class="news-box-content">
-              <p>Uyuni 2021.12 is now available! Key highlights: Salt a as bundle, aarch64 for openSUSE Leap 15.3, CentOS and clones clients, prometheus blackbox exporter and much more! Check the <a href="../pages/stable-version.html#releasenotes">release notes</a> for more details.</p>
+              <p>With regard to the recent publication of the <a href="https://nvd.nist.gov/vuln/detail/CVE-2021-44228" target="_blank">Log4Shell vulnerability</a>,  we can <a href="here-link-to-message-at-the-list-archive" target="_blank">announce</a> that as to our current knowledge Uyuni installations are not affected.</p>
```
Comment (juliogonzalez) [inline index.html:239]:
```suggestion
              <p>With regard to the recent publication of the <a href="https://nvd.nist.gov/vuln/detail/CVE-2021-44228" target="_blank">Log4Shell vulnerability</a>,  we can announce Uyuni installations are not affected.</p>
```

File: pages/news.html [hunk 2]
```diff
@@ -87,7 +87,8 @@ <h1>NEWS</h1>
 <div class="container page-wrapper">
     <div class="container">
         <div class="row content-padding subpage-content">
-            <p><strong>09/12/2021</strong> - Uyuni 2021.12 is now available! Key highlights: Salt a as bundle, aarch64 for openSUSE Leap 15.3, CentOS and clones clients, prometheus blackbox exporter and much more! Check the <a href="../pages/stable-version.html#releasenotes">release notes</a> for more details.</p>
+            <p><strong>13/12/2021</strong> - With regard to the recent publication of the <a href="https://nvd.nist.gov/vuln/detail/CVE-2021-44228" target="_blank">Log4Shell vulnerability</a>,  we can <a href="here-link-to-message-at-the-list-archive" target="_blank">announce</a> that as to our current knowledge Uyuni installations are not affected.</p>
```
Comment (juliogonzalez) [inline pages/news.html:90]:
```suggestion
            <p><strong>13/12/2021</strong> - With regard to the recent publication of the <a href="https://nvd.nist.gov/vuln/detail/CVE-2021-44228" target="_blank">Log4Shell vulnerability</a>,  we can announce that Uyuni installations are not affected.</p>
```

File: pages/news.html [hunk 3]
```diff
@@ -87,7 +87,8 @@ <h1>NEWS</h1>
...
+            <p><strong>09/12/2021</strong> - Uyuni 2021.12 is now available! Key highlights: Salt as a bundle, aarch64 for openSUSE Leap 15.3, CentOS and clones clients, prometheus blackbox exporter and much more! Check the <a href="../pages/stable-version.html#releasenotes">release notes</a> for more details.</p>
```
Comment (juliogonzalez) [inline pages/news.html:91]: This line should not get any changes. Maybe you changed spaces to tabs, or tabs to spaces?

**Review by juliogonzalez — CHANGES_REQUESTED** [review]
Given the mailing list archive is down, I suggest we modify the PR to announce this, without any list to the mailing list.

File: pages/news.html (hunk 2 above)
Comment (juliogonzalez) [reply pages/news.html:90]: If you agree with the changes, just accept them, merge, and I will prepare the tweet.

File: pages/news.html (hunk 3 above)
Comment (renner) [reply pages/news.html:91]: I fixed a typo reported by @vzhestkov (in a separate commit): "Salt a as bundle" vs. "Salt as a bundle"

File: pages/news.html (hunk 2 above)
Comment (renner) [reply pages/news.html]: Both suggestions are considered, I added a new commit.

File: index.html (hunk 1 above)
Comment (renner) [reply index.html]: Thanks, both suggestions are considered, I added a new commit.

**Review by juliogonzalez — APPROVED** [review]

**Pull request merged**